Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -431,43 +520,82 @@ public class EOSSessionsManager : IEOSSubManager
 
         /// <summary>
-        /// Messages with this as the {2} parameter of <see cref="InformSessionOwnerMessageFormat"/> indicate a user is joining the session.
+        /// Messages with this as the {2} parameter of <see cref="InformSessionOwnerMessageFormat"/> indicate a user is joining the Session.
         /// </summary>
         private const string P2P_JOINING_SESSION_MESSAGE_ELEMENT = "JOIN";
 
         /// <summary>
-        /// Messages with this as the {2} parameter of <see cref="P2P_INFORM_SESSION_MESSAGE_FORMAT"/> indicate a user is leaving the session.
+        /// Messages with this as the {2} parameter of <see cref="P2P_INFORM_SESSION_MESSAGE_FORMAT"/> indicate a user is leaving the Session.
         /// </summary>
         private const string P2P_LEAVING_SESSION_MESSAGE_ELEMENT = "LEAVE";
 
         /// <summary>
-        /// Messages with this as the {2} parameter of <see cref="P2P_INFORM_SESSION_MESSAGE_FORMAT"/> indicate that a user should re-acquire and refresh session information.
+        /// Messages with this as the {2} parameter of <see cref="P2P_INFORM_SESSION_MESSAGE_FORMAT"/> indicate that a user should re-acquire and refresh Session information.
+        /// Note that not every element of the Session can be re-acquired this way.
+        /// It is necessary to mirror <see cref="Register(string, ProductUserId)"/>, <see cref="UnRegister(string, ProductUserId)"/>,
+        /// <see cref="StartSession(string)"/>, and <see cref="EndSession(string)"/> using their associated messages.
         /// </summary>
         private const string P2P_REFRESH_SESSION_MESSAGE_ELEMENT = "REFRESH";
 
         /// <summary>
-        /// Messages with this as the {2} parameter of <see cref="P2P_INFORM_SESSION_MESSAGE_FORMAT"/> indicate that the owner of a session has destroyed the session,
-        /// so members of the session should also remove themselves from the session.
+        /// Messages with this as the {2} parameter of <see cref="P2P_INFORM_SESSION_MESSAGE_FORMAT"/> indicate that the Owner of a Session has destroyed the Session,
+        /// so members of the Session should also remove themselves from the Session.
         /// </summary>
         private const string P2P_SESSION_OWNER_DESTROYED_SESSION_MESSAGE_ELEMENT = "DESTROY";
 
+        /// <summary>
+        /// Messages with this as the {2} parameter of <see cref="P2P_INFORM_SESSION_MESSAGE_FORMAT"/> indicate that the Owner of a Session has started the Session.
+        /// Members of the Session should start their Session locally.
+        /// When a user joins a Session, if the Session is <see cref="OnlineSessionState.Starting"/> or <see cref="OnlineSessionState.InProgress"/>,
+        /// the Owner should inform the joining user that the Session should be Started locally.
+        /// Otherwise when the Owner starts a Session, they should inform the users to start theirs locally.
```

Comment:
This is quite a bit of information to include for a `const` definition. It seems like only line `568` is required for the comment. The rest of the content seems important, but seems like it might belong elsewhere.

Similar thoughts regarding the following two `const` definitions.

Reply:
I think I'm comfortable with the idea that Register and such will document this for me. I'll snip all but the first two lines, which I think are necessary, and replace it with a `<see cref="InformUsersOfRegistrationStatusChange(string)"/>`